packages/ubuntu_bootstrap/test/test_utils.dart

```diff
@@ -1,11 +1,15 @@
 import 'package:flutter/material.dart';
 import 'package:flutter_test/flutter_test.dart';
 import 'package:mockito/annotations.dart';
+import 'package:mockito/mockito.dart';
 import 'package:ubuntu_bootstrap/l10n.dart';
 import 'package:ubuntu_bootstrap/services.dart';
+import 'package:ubuntu_provision/ubuntu_provision.dart';
 import 'package:ubuntu_wizard/ubuntu_wizard.dart';
 import 'package:yaru/yaru.dart';
 
+import 'confirm/test_confirm.dart';
```

Comment:
We should probably import `'../../ubuntu_provision/test/test_utils.mocks.dart'` here

Reply:
Ah yes, in the long run we should probably create an internal package in the monorepo with test helpers, so that we don't have to import files like this.